applications/DG-Max/Programs/DGMaxEigen.cpp

```diff
@@ -95,6 +96,78 @@ int main(int argc, char** argv) {
     return 0;
 }
 
+template <std::size_t DIM>
+class DGMaxEigenDriver : public AbstractEigenvalueSolverDriver<DIM> {
+
+   public:
+    DGMaxEigenDriver(KSpacePath<DIM>& path,
+                     std::size_t targetNumberOfEigenvalues)
+        : targetNumberOfEigenvalues_(targetNumberOfEigenvalues),
+          currentPoint_(0),
+          path_(path),
+          frequencyResults_(path.totalNumberOfSteps()) {
+
+        outFile.open("frequencies.csv");
```

Comment:
do you want a hardcoded filename here?

Reply:
It was previously also hardcoded, and for me it's nice to have a fixed value (no searching for the relevant file).

On the cluster I always run the programs in a new (empty) working directory, so no risk to have clashes. So at the moment I would say that it would be more work to make it variable, for something that won't be used anywhere soon.